apps/win32/src/reports/appium.txt

```diff
@@ -0,0 +1,646 @@
+[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
```

Comment:
I'm guessing we don't want these reports checked in?

Reply:
Nope, removing now

Reply:
Might want to get them in the gitignore if you haven't already

Reply:
you should add the whole reports folder to gitignore because the test xmls will go here too

you will also want to publish the xmls in the ado yaml (or use `run_wdio` to parse them and print out the test failures in the CI console)

Reply:
Added to gitignore.